### PR TITLE
[ready] Make tab and trailing white space visible

### PR DESCRIPTION
**Pull request by dunghuynh**
1. Make tabs visible
2. Make trailing whitespace visible
Screen Shot 2019-07-15 at 16 37 52
Screen Shot 2019-07-15 at 16 36 21

### CODE REVIEW

File: janus/vim/vimrc [hunk 1]
```diff
@@ -206,6 +206,15 @@ nmap <silent> <Leader>h :<C-U><C-R>=printf("Leaderf! rg -e %s ", expand("<cword>
 
 " System Settings  ----------------------------------------------------------{{{
   set noswapfile
+  " List chars
+  set list                          " Show invisible characters
+  set listchars=""                  " Reset the listchars
```
Comment (tannauit) [inline janus/vim/vimrc]: Why do we need to reset and set a new one below?

Comment (dunghuynh) [reply janus/vim/vimrc]: We just don't know if listchars has been set anywhere else. Resetting it to ensure we start from scratch

**Comment by tiendo1011** [comment]
Can u update the PR description about why would we need this a @dunghuynh?

**Comment by phuongnd08** [comment]
I don't think I have any special configuration yet I already have this on my nvim
image

**Comment by tiendo1011** [comment]
Same here.

**Comment by dunghuynh** [comment]
This is not just for trailing spaces.
This is to show `tabs` as well.

**Pull request merged**